sdks/python/tox.ini

```diff
@@ -329,3 +329,15 @@ commands =
   # Run all PyTorch unit tests
   # Allow exit code 5 (no tests run) so that we can run this command safely on arbitrary subdirectories.
   /bin/sh -c 'pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_pytorch {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
+
+[testenv:py{37,38,39,310}-tf-{211}]
```

Comment:
I think you would also need to add tox info similar to this https://github.com/apache/beam/blob/b90ed15146791e6dd04b468250acb4f406cf49f8/sdks/python/test-suites/tox/py38/build.gradle#L125

Reply:
I think this was done so that the time of python precommits won't increase(it was used to run for 90 minutes)

Reply:
okay, thanks!

Reply:
yeah. I missed running PostCommit on this PR.